src/pipx/package_specifier.py

```diff
@@ -36,6 +44,16 @@ def _split_path_extras(package_spec: str) -> Tuple[str, str]:
         return (package_spec, "")
 
 
+def _check_package_path(package_path: str) -> Tuple[Path, bool]:
+    pkg_path = Path(package_path)
+    try:
+        pkg_path_exists = pkg_path.exists()
```

Comment:
I don’t think `exists()` raise `OSError`?

Reply:
I moved these few lines into a new function:

https://github.com/pypa/pipx/blob/fc9919d9da647ead6ee83fa9bde4879eb13a7517/src/pipx/package_specifier.py#L75-L78

Reply:
But actually it only returns True or False.

Reply:
https://docs.python.org/3/library/pathlib.html#methods

> Changed in version 3.8: [exists()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.exists), [is_dir()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_dir), [is_file()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_file), [is_mount()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_mount), [is_symlink()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_symlink), [is_block_device()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_block_device), [is_char_device()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_char_device), [is_fifo()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_fifo), [is_socket()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_socket) now return False instead of raising an exception for paths that contain characters unrepresentable at the OS level.

Reply:
Looks like it is: https://docs.python.org/3/library/pathlib.html#pathlib.Path.exists